client/app/widgets/ToggleLocale/__tests__/ToggleLocale.spec.jsx

```diff
@@ -17,23 +22,25 @@ describe('ToggleLocale', () => {
   });
 
   it('does nothing if the previous locale is the same as the selected', () => {
-    const wrapper = mount(component);
+    render(component);
     act(() => {
```

Comment:
`act` is not needed here 👍

See [this point](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily).

Reply:
Good call! I didn't think about it since I think it is not really related to the migration.